.swift-version

```diff
@@ -1 +1 @@
-3.1
+4.2
```

Comment:
We should probably get rid of this file entirely, and add the `swift_version` to our Podspec instead.